packages/apps/proof-of-us/src/components/AvatarEditor/AvatarEditor.tsx

```diff
@@ -106,8 +115,27 @@ export const AvatarEditor: FC<IProps> = ({ next }) => {
     next();
   };
 
+  const handleClose = async () => {
+    if (!confirm('Are you sure you want to stop with this token?')) return;
+    if (proofOfUs) await closeToken({ proofOfUsId: proofOfUs.proofOfUsId });
+    setIsMounted(false);
+    router.replace('/user');
+  };
+
   return (
     <section className={wrapperClass}>
+      <div className={headerClass}>
```

Comment:
something wrong with the size of the camera now
<img width="867" alt="image" src="https://github.com/kadena-community/kadena.js/assets/4015521/ba27d50d-23ae-4c30-9c41-053ef41fb120">